package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "base",
   "private": true,
-  "version": "6.20.1",
+  "version": "6.20.3",
```

Comment:
I forgot to bump the version for the last hotifx. I was rushing.

Reply:
It happens. Just remember for next time.